sanctuarycomputer/contentful-fragment
# Prevent undefined schema error when setting up schema from the UI

To test:
In `app/application.js`
line 5:
```
const DEV = true;
```
line 107-109
```
// instance: {
//      schemaShorthand: "Event Location:Symbol,Event Url: Symbol,Event Date:Date,Logo:Blob"
// }
```
line 112
```
  _value: { }, // DUMMY_DATA,
```

Then, in your terminal
```
yarn
ember serve
```

Navigate to localhost:4200

Expect to be able to setup your fragment schema with the UI as normal.
If you change back lines 107-109 and line 112, expect to see the demo with a predefined schema once again.
